lib/checkunusedfunctions.cpp

```diff
@@ -382,8 +433,10 @@ void CheckUnusedFunctions::analyseWholeProgram(ErrorLogger * const errorLogger,
             continue;
 
         if (calls.find(functionName) == calls.end()) {
-            const Location &loc = decl->second;
-            unusedFunctionError(errorLogger, loc.fileName, loc.lineNumber, functionName);
+            if (!isOperatorFunction(functionName)) {
```

Comment:
This code is so short, so no early continue is needed.. keep this code as it is.